install/kubernetes/cilium/charts/hubble/templates/daemonset.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: hubble
+  namespace: {{ .Release.Namespace }}
```

Comment:
As mentioned in https://github.com/cilium/cilium/pull/10648#issuecomment-606055294 we are deprecating this DaemonSet as of Cilium 1.8. Could you remove the `hubble` DaemonSet and all related values instead make sure the chart works with embedded Hubble? The config map for embedded Hubble is here (based on #10794):

https://github.com/cilium/cilium/blob/95f492f25d8ea7a1585503b36683eb690be40b5b/install/kubernetes/cilium/charts/config/templates/configmap.yaml#L363-L387

I think most of the work for the Hubble server itself is already done. The CRI metadata is not implemented in embedded Hubble, so that these values can be removed. I think the biggest change is extending the `hubble-listen-addresses` in the ConfigMap to add a listener on port 50051 when `.Values.ui.enabled` is `true`, similar to how this DaemonSet currently does it.

Please do not hesitate to reach out to either me other other folks in `#hubble-devel` on Slack if you have questions or need some help or pointers. Thanks!

Reply:
Sure will address your comments and will contact you for any questions.

Reply:
What do you think. Should I continue with this PR or drop it?

Reply:
It's up to you. We do want to have the UI and other components part of the Helm chart eventually, so adapting this PR does make sense. Feel free to ping me privately if you need assistance!

Reply:
> It's up to you. We do want to have the UI and other components part of the Helm chart eventually, so adapting this PR does make sense. Feel free to ping me privately if you need assistance!

Sure will check with you.

Reply:
> It's up to you. We do want to have the UI and other components part of the Helm chart eventually, so adapting this PR does make sense. Feel free to ping me privately if you need assistance!

@gandro  So in this case do you want me to completely remove the daemonset.yaml and incorporate the changes in the config/templates/configmap.yaml or want to create a configmap.yaml in hubble/templates/configmap,yaml and configure everything there. Can you confirm.

Reply:
> @gandro So in this case do you want me to completely remove the daemonset.yaml and incorporate the changes in the config/templates/configmap.yaml or want to create a configmap.yaml in hubble/templates/configmap,yaml and configure everything there. Can you confirm.

Kind of. So the way I imagine this is that there is not seperate `hubble` chart anymore. Only one for `hubble-ui`. `hubble` itself now lives inside cilium-agent. Therefore, all Hubble related values are set in ` config/templates/configmap.yaml`. I think the version you pushed now seems to go into the right direction :tada:  I will do a more in depth review soon!